### PR TITLE
[wil] Updating WIL version to match its nuget package version

### PR DESCRIPTION
**Pull request by anaaru**
Updating wil vcpkg to match wil.nupkg version which is 1.20210204.0
Removed the fix-install-headers.patch since its no longer needed.

### CODE REVIEW

File: ports/wil/CONTROL [hunk 1]
```diff
@@ -1,5 +1,5 @@
 Source: wil
-Version: 2020-05-19
+Version: 2021-02-04
 Port-Version: 1
```
Comment (Hoikas) [inline ports/wil/CONTROL:3]:
```suggestion
```

Comment (anaaru) [reply ports/wil/CONTROL:3]: Any particular reason to remove line-3?

Comment (Hoikas) [reply ports/wil/CONTROL:3]: When a port is updated to a new version, the port-version should be reset. See the [documentation](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/control-files.md#port-version) for more details. The general practice is to delete the entire line.

Comment (anaaru) [reply ports/wil/CONTROL]: Addressed in next iteration.

Comment (JonLiu1993) [reply ports/wil/CONTROL:3]: Hi @anaaru ,Thanks for posting this issue, @Hoikas gave a good suggestion, please modify the file before I can merge

**Review by JonLiu1993 — COMMENTED** [review]
@Hoikas gave a good suggestion, please modify the file before I can merge

**Review by JonLiu1993 — APPROVED** [review]

**Comment by ras0219-msft** [comment]
Thanks!

**Pull request merged**